Integrations/QRadar/QRadar.yml

```diff
@@ -1,6 +1,6 @@
 category: Analytics & SIEM
 commonfields:
-  id: QRadar
+  id: IBM QRadar 5.5
```

Comment:
Don't change the ID

Reply:
the original ID was IBM QRadar,
Do you want me to remove 5.5 ?